app/cdap/components/PipelineTriggers/EnabledTriggersTab/EnabledCompositeTriggerRow.tsx

```diff
@@ -101,31 +101,27 @@ const EnabledGroupTriggerRowView = ({
       square
       expanded={isExpanded}
       onChange={() => handleAccordionClick()}
-      data-cy={isExpanded ? `${triggerGroup.name}-expanded` : `${triggerGroup.name}-collapsed`}
+      data-cy={
```

Comment:
Could you add `data-testid` as well while you're at it? `data-cy` will be deleted upon retiring cypress.

Reply:
got it, updated